rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/server/NSCreationServlet.java

```diff
@@ -96,44 +102,58 @@ private void createNs(HttpServletRequest request, HttpServletResponse response)
             List<?> fileItems = upload.parseRequest(request);
             Iterator<?> i = fileItems.iterator();
             while (i.hasNext()) {
-                FileItem fi = (FileItem) i.next();
-                String fieldName = fi.getFieldName();
-                if (fi.isFormField()) {
-                    if (fieldName.equals("sessionId")) {
-                        sessionId = fi.getString();
-                    } else if (fieldName.equals("nsCallback")) {
-                        callbackName = fi.getString();
-                    } else if (fieldName.equals("nsName")) {
-                        nsName = fi.getString();
-                    } else if (fieldName.equals("nodesRecoverable")) {
-                        nodesRecoverable = fi.getString();
-                    } else if (fieldName.equals("deploy")) {
-                        if (fi.getString().equals(Boolean.TRUE.toString())) {
+                FileItem formField = (FileItem) i.next();
+                String formFieldName = formField.getFieldName();
+                if (formField.isFormField()) {
+                    String formFieldValue = formField.getString();
+                    if (formFieldName.equals("sessionId")) {
```

Comment:
You can put sessionId, nsCallback, .... in constants,
but this is not really important

Reply:
In fact, there are lots of situations like that, it can be useful one day to create a helper or utility class to store all UI&co element names...
So don't mind my comment